tests/foreman/ui_airgun/test_bookmark.py

```diff
@@ -0,0 +1,130 @@
+import random
+
+from airgun.session import Session
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo.constants import BOOKMARK_ENTITIES
+from robottelo.decorators import bz_bug_is_open, fixture, rm_bug_is_open, tier2
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def custom_user(module_org):
+    """Custom user with viewer role for tests validating 'public' bookmark
+    option.
+    """
+    viewer_role = entities.Role().search(query={'search': 'name="Viewer"'})[0]
+    bm_role = entities.Role().create()
+    entities.Filter(
+        permission=entities.Permission(
+            resource_type='Bookmark').search(
+            filters={'name': 'view_bookmarks'}),
+        role=bm_role,
+    ).create()
```

Comment:
Why we need this new role with view_bookmarks, is viewer_role not enough (it does also contain Bokmark::view_bookmarks) ?

Reply:
It was not able to navigate to bookmarks page when i was writing the test so that's why i've added it explicitly. Just retested - now it works ok. No idea what was causing that, thx, will remove that extra filer and role.